content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -116,6 +115,13 @@ In this example:
     NAME                          DESIRED   CURRENT   READY   AGE
     nginx-deployment-75675f5897   3         3         3       18s
     ```
+    ReplicaSet output shows the following fields:
+    
+      * `DESIRED` displays the desired number of _replicas_ of the application, which you define when you create the Deployment. This is the _desired state_.
+      * `CURRENT` displays how many replicas are currently running.
+      * `READY` displays how many replicas of the application are available to your users.
+      * `AGE` displays the amount of time that the application has been running.
```

Comment:
I would add `NAME` as well. Could you revise to include that @rmohta? If you know how to squash your commits, please do that too.

Reply:
Done. Commits have also been squashed.